lib/tilex_web/views/shared_view.ex

```diff
@@ -2,11 +2,20 @@ defmodule TilexWeb.SharedView do
   use TilexWeb, :view
 
   def display_date(post) do
-    Timex.format!(post.inserted_at, "%B %-e, %Y", :strftime)
+    post.inserted_at
+    |> datetime_in_zone()
+    |> Timex.format!("%B %-e, %Y", :strftime)
   end
 
   def rss_date(post) do
-    Timex.format!(post.inserted_at, "%a, %d %b %Y %H:%M:%S GMT", :strftime)
+    post.inserted_at
+    |> datetime_in_zone()
+    |> Timex.format!("%a, %d %b %Y %H:%M:%S GMT", :strftime)
+  end
+
+  defp datetime_in_zone(datetime) do
+    timezone = Timex.Timezone.get("America/Chicago", datetime)
```

Comment:
This seems really weird. You should either use configuration or (even better) local timezone. So I would suggest, that this line should read
```
timezone = Timex.Timezone.local()
```